instrumentation/struts-2.3/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/struts2/ActionInvocationAdvice.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.struts2;
+
+import static io.opentelemetry.javaagent.instrumentation.struts2.Struts2Tracer.TRACER;
+
+import com.opensymphony.xwork2.ActionInvocation;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.javaagent.instrumentation.api.Java8BytecodeBridge;
+import net.bytebuddy.asm.Advice;
+
+public class ActionInvocationAdvice {
+
+  @Advice.OnMethodEnter(suppress = Throwable.class)
+  public static void onEnter(
+      @Advice.This ActionInvocation actionInvocation,
+      @Advice.Local("otelSpan") Span span,
+      @Advice.Local("otelScope") Scope scope) {
+    span = TRACER.startSpan(actionInvocation);
+    scope = TRACER.startScope(span);
+
+    TRACER.updateServerSpanName(Java8BytecodeBridge.currentContext(), actionInvocation.getProxy());
```

Comment:
We [have recently switched](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1540) to using `tracer()`